unison-cli/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -2103,13 +2103,12 @@ ioTest :: InputPattern
 ioTest =
   InputPattern
     "io.test"
-    []
+    ["test.io"]
     I.Visible
-    []
+    [(Required, exactDefinitionTermQueryArg)]
     ( P.wrapColumn2
         [ ( "`io.test mytest`",
-            "Runs `!mytest`, where `mytest` is searched for in the most recent"
-              <> "typechecked file, or in the codebase."
+            "Runs `!mytest`, where `mytest` is a test that requires the IO ability and has been added to the codebase."
```

Comment:
Does it have to require the `IO` ability for `test.io`/`io.test` to work?

Reply:
I thought that it did at first, but I guess the problem there was that it wasn't a delayed program. It looks like `io.test` works for things that don't declare `IO` or `Exception`, so this wording isn't technically correct. I'll update it.